.travis/klee.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash -x
+# Make sure we exit if there is a failure
+set -e
+: ${SOLVERS?"Solvers must be specified"}
+
+source ${SRC_DIR}/.travis/llvm_compiler.sh
+
+###############################################################################
+# Clone KLEE source code
+###############################################################################
+git clone https://github.com/klee/klee.git ${SRC_DIR}/klee
+
+###############################################################################
+# Testing utils for KLEE
+###############################################################################
+source ${SRC_DIR}/.travis/testing-utils.sh
```

Comment:
@andreamattavelli This is wrong. `testing-utils.sh` calls `exit 0` which means we don't build KLEE. You can see this in the output of the latest build --> https://travis-ci.org/klee/klee-uclibc/jobs/214164351